pkg/testmachinery/argo/argo.go

```diff
@@ -102,7 +102,7 @@ func getWorkflowAffinity() *corev1.Affinity {
 		NodeAffinity: &corev1.NodeAffinity{
 			PreferredDuringSchedulingIgnoredDuringExecution: []corev1.PreferredSchedulingTerm{
 				{
-					Weight:     100,
+					Weight: 100,
```

Comment:
Hmm, do we have different formatters?

Reply:
ah yeah could be. I use `go fmt`

Reply:
I think goland is correct in this case. Could it be that you run an old version of gofmt? See e.g. https://github.com/golang/go/issues/26497

Reply:
Well, just checked the latest gofmt (1.12.7) and it also wrongly formats the above...

Reply:
Nevermind, then it is like it is and we stick to go fmt :)